index.js

```diff
@@ -0,0 +1,75 @@
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
you could add an IIFE also but looks nice that whole code is wrapped into DOMContentLoaded

Reply:
That's right, thanks for bringing it up